### ISSUE TITLE
RNA NER

### ISSUE THREAD

**Issue by juanmirocks** [ashishbaghudana/mthesis-ashish#8]
- [x] Train model with GENIA corpus
- [x] Test performance (with test set or otherwise CV)
- [ ] 2x confirm that training code for GnormPlus is not available
- [ ] Try training with [Moara](http://moara.dacya.ucm.es/suppl_material_bmc.html)
